code/modules/uplink/uplink_item.dm

```diff
@@ -1230,7 +1237,7 @@ var/list/uplink_items = list() // Global list so we only initialize this once.
 	while(crate_value)
 		var/category = pick(uplink_items)
 		var/item = pick(uplink_items[category])
-		var/datum/uplink_item/I = uplink_items[category][item]
+		var/datum/uplink_item/I = [category][item]
```

Comment:
Is there a reason for this change? It's causing the build to fail, you should change it back to `uplink_items[category][item]`.